File: backend/manga-ln-app/src/main/java/com/example/simeon/manga_ln_app/dto/CommentBetaDTO.java
package com.example.simeon.manga_ln_app.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class CommentBetaDTO {

    @NotBlank
    private String authorName;

    @NotNull
    private int chapterId;

    @NotBlank
    private String text;
}

File: mobile-app/app/src/main/java/com/example/manga_ln_app/data/dto/CommentBetaDto.kt
package com.example.manga_ln_app.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CommentBetaDto(
    @SerialName("id") val id: Int,
    @SerialName("author") val author: String,
    @SerialName("text") val text: String
)
